packages/design-system-react-native/src/components/TextButton/TextButton.constants.ts

```diff
@@ -0,0 +1,43 @@
+import { IconSize, IconColor } from '../Icon';
+import { TextColor, TextVariant, FontWeight } from '../Text';
+import type { TextButtonProps } from './TextButton.types';
+
+// Defaults
+export const DEFAULT_TEXTBUTTON_PROPS: Required<
+  Pick<
+    TextButtonProps,
+    | 'textProps'
+    | 'isLoading'
+    | 'loadingText'
+    | 'spinnerProps'
+    | 'startIconProps'
+    | 'endIconProps'
+    | 'onPress'
+    | 'isDisabled'
+    | 'isInverse'
+  >
+> = {
+  textProps: {
+    variant: TextVariant.BodyMd,
+    fontWeight: FontWeight.Medium,
+    color: TextColor.PrimaryDefault,
+  },
+  isLoading: false,
+  loadingText: 'Loading',
+  spinnerProps: {
+    color: IconColor.IconDefault,
+  },
+  startIconProps: {
+    size: IconSize.Sm,
+    testID: 'start-icon',
+  },
+  endIconProps: {
+    size: IconSize.Sm,
+    testID: 'end-icon',
+  },
+  onPress: () => {
+    console.log('Button pressed');
+  },
```

Comment:
We shouldn't use a console.log as default props. You could consider using a no-op function (`() => {}`) or removing the default `onPress` handler entirely.